03_triggers/01_binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: vote-app-pipelinebinding
+spec:
+  params:
+  - name: gitrevision
+    value: $(body.head_commit.id)
+  - name: namespace
+    value: pipelines-tutorial
```

Comment:
can this be obtained somehow?

Reply:
We should put this in the template.
Use sed to update Namespace there.

Reply:
@praveen4g0 need to address this

Reply:
True, I agree +1, but do we make changes as per khurram comment??

Reply:
yes, remove the params .. and use only the one in template

Reply:
Removed the namespace param from trigger-binding, and used sed to update namespace there